ui/src/pages/organizations/add.tsx

```diff
@@ -77,24 +76,23 @@ const validationSchema = Yup.object().shape({
 });
 
 export default () => {
-  const dispatch = useDispatch();
+  const [createOrg] = useAsyncThunk(createOrganization, {
```

Comment:
`createOrg` and `createOrganization` are too similar to unambiguously tell what they mean. Please choose better names.